dev-docs/publisher-api-reference.md

```diff
@@ -445,6 +446,14 @@ Use this method to get all of the bids that have won their respective auctions b
 
 <hr class="full-rule">
 
+<a name="module_pbjs.getMappingFileInfo"></a>
+
+### pbjs.adserver.freewheel.getMappingFileInfo() ⇒ Object
```

Comment:
It should be pbjs.adserver.freewheel.getTargeting()

You can refer to this http://prebid.org/dev-docs/publisher-api-reference.html#module_pbjs.adServers.dfp.buildVideoUrl

Reply:
Revised doc, please review for accuracy.